README.md

```diff
@@ -1,19 +1,122 @@
----
-
 # Extended SplashScreen
-
-{Project tag line}
-
-{Small description of the purpose of the project}
+Customizable async Extended splashscreen recipe for MVVM development.
```

Comment:
I think the description needs to be reworded. In my opinion, it should explain its actual uses. Like, what does it achieve exactly? Even why it's necessary to have this?